docs/src/modules/components/AdDisplay.js

```diff
@@ -23,6 +23,18 @@ const Root = styled('span', { shouldForwardProp: (prop) => prop !== 'shape' })((
 export default function AdDisplay(props) {
   const { ad, className, shape = 'auto' } = props;
 
+  React.useEffect(() => {
+    // Avoid an exceed on the Google Analytics quotas.
+    if (Math.random() < 0.9 || !ad.label) {
+      return;
+    }
```

Comment:
Can you explain more what's this for?

Reply:
Not sure about which aspect is unclear.

The early return is just a way to send only 10% of the ads display event to Goggle Analitics. A similar approach is used for top page ads

Reply:
I added a commit to make sure this ratio is consistent between all ads